src/components/vertical-divider/vertical-divider.spec.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import VerticalDivider from '.';
+import { StyledVerticalWrapper, StyledDivider } from './vertical-divider.style';
+import { assertStyleMatch, testStyledSystemSpacing } from '../../__spec_helper__/test-utils';
+import { baseTheme } from '../../style/themes';
+
+function render(props = {}) {
+  return mount(<VerticalDivider { ...props } />);
+}
+
+describe('VerticalDivider', () => {
+  describe('styles', () => {
+    testStyledSystemSpacing(
+      props => <VerticalDivider { ...props } />, {
+        pt: '24px', pb: '24px', pl: '24px', pr: '24px'
```

Comment:
Don't need to change this but just FYI, you can pass these in as `pt: 3` etc as well. Although with my above comment you'll just need `p: 3` in here!